test/SourceKit/Indexing/indexstore.swift

```diff
@@ -0,0 +1,17 @@
+// RUN: %empty-directory(%t)
+// RUN: %sourcekitd-test -req=index-to-store %s -index-store-path %t/idx -index-unit-output-path %t/indexstore.o -- -c -module-name indexstoremodule -file-prefix-map %S=REMAPPED_SRC_ROOT -file-prefix-map %t=REMAPPED_OUT_DIR %s
```

Comment:
Could you also add a test without any prefix map, with the output path specified in the regular args as well, and with multiple files specified (which IIRC we hack the frontend args to then just have the one primary file based on the requested file)?

Reply:
Done. Is there anything else I should do to test this out?

Reply:
Sorry for the delay!

There's input-unit-output-file as an argument as well. But other than that... I don't think so. The tests you have seem reasonable to me 🙇‍♂️